.github/workflows/update-robots-txt.yml

```diff
@@ -0,0 +1,88 @@
+name: Update robots.txt
+
+permissions: write-all
+
+on: 
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  sync-upstream:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Create Branch
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        uses: peterjgrainger/action-create-branch@v2.2.0
+        with:
+          branch: robots.txt-update
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: robots.txt-update
+          fetch-depth: 0 
+      - name: Update robot.txt
+        id: update
+        env:
+          API_KEY: ${{ secrets.ROBOTS }}
+        run: |
+          '#       _---~~(~~-_.
+          #     _{        )   )
+          #   ,   ) -~~- ( ,-' )_
+          #  (  `-,_..`., )-- '_,)
+          # ( ` _)  (  -~( -_ `,  }
+          # (_-  _  ~_-~~~~`,  ,' )
+          #   `~ -^(    __;-,((()))
+          #         ~~~~ {_ -_(())
+          #                `\  }
+          #                  { }  
+          # BRAAAAAAAAIIIINNSSSSSSS
+
+          ' >> public/robots.txt
```

Comment:
This is important

Reply:
Could we add something like http://www.patorjk.com/software/taag/#p=display&f=Chiseled&t=Brains
I think it might be important for reliability

Reply:
yaya

Reply:
ok, how's that? :D